JS/Lesson35/onder/my-app/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext, useState } from "react";
+import { NavLink } from "react-router-dom";
+import "./styles.css";
+import { GlobalContext } from "../../Context/GlobalState";
+
+const Navbar = () => {
```

Comment:
Add Context for the user state to the application, then in navbar hide an show certain pages depending on weather user is logged in or not.